src/Calculator.java

```diff
@@ -1,21 +1,142 @@
+import java.util.Scanner;
+
 public class Calculator {
 
-    // global variables
-    private static final Scanner scanner = new Scanner(System.in);
+	// global variables
+	private static final Scanner scanner = new Scanner(System.in);
+
+	private static double square(double a) {
+		return a * a;
+	}
+
+	public static double pythagoras(double a, double b) {
+		return Math.sqrt(square(a) + square(b));
+	}
+
+	public static void calculate(String calcMethod, String numberOne, String numberTwo, String choice)
+			throws NumberFormatException {
```

Comment:
Why do you throw an exception? For example, the `Integer.parseInt()` method does throw one anyways.